scam-detector-py/src/agent.py

```diff
@@ -1361,9 +1474,20 @@ def handle_block(block_event: forta_agent.block_event.BlockEvent) -> list:
             persist_state()
             logging.info(f"{BOT_VERSION}: Persisted state")
             
-            
-            
-        
+        # Create reactive FP findings
+        if REACTIVE_LIKELY_FPS:
+            # Check 5 addresses per block
+            addresses_to_check = list(REACTIVE_LIKELY_FPS.keys())[:5]
+            for address in addresses_to_check:
+                if Utils.is_fp(w3, address, CHAIN_ID):
```

Comment:
why not do this in update_reactive_likely_fps. This way, you would store less addresses (the actual FPs as opposed to all scammer addresses) in the dict?

Reply:
im fine with doing 5 each, but I am saying, why not check whether its an FP in the update function.

Reply:
@christian-forta , I moved this part into the update function, but `REACTIVE_LIKELY_FPS` still need to be stored before checking each for being an FP, as we're fetching e.g. 140 labels but only processing 5 of them per block. Am I misunderstanding your question?